admin/system/news-save.php

```diff
@@ -0,0 +1,26 @@
+<?php
+    include_once("connection.php");
+
+    $judul = $_POST['judul'];
```

Comment:
Codacy found an issue: [Detected usage of a possibly undefined superglobal array index: $_POST['judul']. Use isset() or empty() to check the index exists before using it](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)